client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -0,0 +1,179 @@
+<template>
+  <div>
+    <div
+      v-for="(rule, index) in rules"
+      :key="`rule-${index}`"
+    >
+      <b-row class="justify-content-between">
+        <b-col cols="6">
+          <h5>{{ rule.label }}</h5>
+          <b-form-checkbox
+            v-model="rule.strict"
+            class="mt-3"
+            name="allow"
+            switch
+          >
+            {{ $t('allowRecordsSave') }}
+          </b-form-checkbox>
+        </b-col>
+      </b-row>
+      <div class="mt-3">
+        <b-form-input
+          :placeholder="$t('warningMessage')"
+        />
+        <div class="table-responsive">
+          <table class="table table-borderless">
+            <thead>
+              <tr>
+                <th colspan="2" />
+                <th style="width: 250px;">
+                  {{ $t('valueModifiers') }}
+                </th>
+                <th style="width: 250px;">
+                  {{ $t('multiValues') }}
+                </th>
+              </tr>
+            </thead>
+            <tbody v-if="rule.constraints">
+              <tr
+                v-for="(constraint, consIndex) in rule.constraints"
+                :key="`constraint-${consIndex}`"
+              >
+                <td>{{ constraint.attribute }}</td>
```

Comment:
This should probably be the field.label || field.name

Reply:
![image](https://user-images.githubusercontent.com/23015247/210577837-f928676f-fdb3-4fbe-89c0-e7dcd358cc5b.png)
This was the structure of the constraints on the backend that's why I used `attribute`

Reply:
That is true, it should submit attribute but for user it should display `field.label || field.name`.